Gauge.CSharp.Lib.UnitTests/ScreenshotTests.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015 ThoughtWorks, Inc.
+//
+// This file is part of Gauge-CSharp.
+//
+// Gauge-CSharp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Gauge-CSharp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Gauge-CSharp.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using NUnit.Framework;
+
+namespace Gauge.CSharp.Lib.UnitTests
+{
+    [TestFixture]
+    public class DefaultScreenshotTests
+    {
+        [Test]
+        public void DefaultScreenshotFileTest ()
+        {
+            Environment.SetEnvironmentVariable("screenshots_dir", "data");
+            var defaultScreenshotWriter = new DefaultScreenshotWriter();
```

Comment:
This will try to capture an actual screenshot. It starts a process.
Try to mock it